libzhlgen/test/functions/ELF_amd64/1.6.13/MenuScreen.zhl

```diff
@@ -4,3 +4,5 @@ cleanup __amd64 void MenuScreen::constructor(MenuScreen *this);
 cleanup __amd64 void MenuScreen::OnRender(MenuScreen *this);
 ".415641554989fd488d7f7841":
 cleanup __amd64 void MenuScreen::Open(MenuScreen *this);
+"415741564155415455534889fb4883ec78488b5720":
+cleanup __amd64 void MenuScreen::OnLanguageChange(MenuScreen *this);
```

Comment:
1. Missing newline at end of file.
2. Please find the appropriate location for this `OnLanguageChange` is it defined after `Open` in the executable? And use `.` for the match to not restart matching from the top of the executable and instead to continue after the previous match. (there should ideally only be one non-relative match per ZHL file, and that being the first signature)